dts/arm/st/f4/stm32f4.dtsi

```diff
@@ -281,6 +283,13 @@
 				status = "disabled";
 				#pwm-cells = <3>;
 			};
+
+			qdec {
+				compatible = "st,stm32-qdec";
+				status = "disabled";
+				st,input-filter-level = <NO_FILTER>;
+				st,counts-per-revolution = <10>;
```

Comment:
why is this set at dts/ level? Seems like a mechanical property of the encoder, right?

Reply:
so, if I understand correctly, you say it's better to keep this on the board's DTS/overlay only, right?

Reply:
What I'd do:

- `st,input-filter-level`: set its default in the binding to `NO_FILTER` (justification: SoC default). Adding it to `dts/` it's de-facto setting a default. If not, force end user (ie board) to make the choice
- `st,counts-per-revolution`: force end user since it's a mechanical property of the encoder they'll use